dev/sg/sg_ci.go

```diff
@@ -226,7 +229,26 @@ Note that Sourcegraph's CI pipelines are under our enterprise license: https://g
 					}
 				}
 
-				stdout.Out.WriteLine(output.Linef("", output.StylePending, "Requesting build for branch %q at %q...", branch, commit))
+				var rt runtype.RunType
+				if len(args) == 0 {
+					rt = runtype.PullRequest
+				} else {
+					rt = runtype.Compute("", fmt.Sprintf("%s/", args[0]), nil, runtype.RunTypeFilter{PrefixOnly: true})
```

Comment:
I think we can compute with the generated branch instead, then we won't need `RunTypeFilter`:

```suggestion
					rt = runtype.Compute("", fmt.Sprintf("%s/%s", args[0], branch), nil)
```

Reply:
Updated in the latest